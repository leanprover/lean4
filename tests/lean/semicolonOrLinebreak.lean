def f (y : Nat) :=
  let x := 1x + y -- Error

structure Point where
 x : Nat
 y : Nat
deriving Repr

def mkPoint1 (a : Nat) : Point where
  x := 1y := a -- Error

def mkPoint2 (a : Nat) : Point where
  x := 1
  y := a

def mkPoint3 (a : Nat) : Point where
  x := 1; y := a

def mkPoint?? (x y : Nat) : Point :=
  { x y }
