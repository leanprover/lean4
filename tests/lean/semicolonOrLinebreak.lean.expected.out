semicolonOrLinebreak.lean:2:12: error: expected ';' or line break
semicolonOrLinebreak.lean:9:31-10:8: error: field 'y' is missing
semicolonOrLinebreak.lean:10:8: error: expected command
semicolonOrLinebreak.lean:20:8: error: expected ':'
